err_stash.py

```diff
@@ -162,6 +162,13 @@ def fetch_repo_commits(
         return repo.compare(to_branch, from_branch).commits
 
 
+@attr.s()
```

Comment:
Try to make it read-only. 👍

Reply:
`@attr.s(frozen=True)` does the job! it is right?